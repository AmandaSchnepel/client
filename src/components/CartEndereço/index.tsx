import * as S from './styles'

const CartEndereço = () => (
  <S.Wrapper>
    <h1>CartEndereço</h1>
  </S.Wrapper>
)

export default CartEndereço
