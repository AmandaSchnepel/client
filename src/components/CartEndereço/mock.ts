export default [
  {
    title: 'Rua Pará 1179 - 83040010',
    subtitle: 'Boneca do iguaçu',
    tempo: '30 - 40 min'
  }
]
