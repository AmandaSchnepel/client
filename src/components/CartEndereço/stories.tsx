import { Story, Meta } from '@storybook/react'
import CartEndereço from '.'

export default {
  title: 'CartEndereço',
  component: CartEndereço
} as Meta

export const Default: Story = () => <CartEndereço />
