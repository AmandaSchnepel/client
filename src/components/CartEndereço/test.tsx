import { render, screen } from '@testing-library/react'

import CartEndereço from '.'

describe('<CartEndereço />', () => {
  it('should render the heading', () => {
    const { container } = render(<CartEndereço />)

    expect(
      screen.getByRole('heading', { name: /CartEndereço/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
