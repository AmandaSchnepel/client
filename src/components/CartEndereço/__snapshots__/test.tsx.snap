// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CartEndereço /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    CartEndereço
  </h1>
</main>
`
